sdk/lusid/configuration.py

```diff
@@ -11,16 +9,13 @@
 """
 
 
-from __future__ import absolute_import
-
 import copy
 import logging
 import multiprocessing
 import sys
 import urllib3
 
-import six
-from six.moves import http_client as httplib
+from http import client as http_client
```

Comment:
Do you know why these diffs exist? I didn't expect there to be any diffs except where we modified the default template, but it looks like the SDK we have differs even from the default template?

Reply:
So I notice that in our [lusid-sdk-generator](https://github.com/finbourne/lusid-sdk-generators) project, no configuration.mustache template existed for me to edit. So I added the latest one from the [openapi-generator](https://github.com/OpenAPITools/openapi-generator) project. When I ran the generation script, these diffs appear. Is this the proper procedure, or how do I go about this?

Reply:
Update the mustache template to be the same version as the generator we use as discussed in the generator repo